js/doc-snippets/src/flows/index.ts

```diff
@@ -67,7 +67,10 @@ export const menuSuggestionFlowMarkdown = ai.defineFlow(
   {
     name: 'menuSuggestionFlow',
     inputSchema: z.string(),
-    outputSchema: z.string(),
+    outputSchema: z.object({
```

Comment:
No... the flow output type is `z.string()`... see line 84:
```ts
return `**${output.dishname}**: ${output.description}`
```

Reply:
Oh whoops. Dunno how I got confused when reading docs.